src/en/help-aws.md

```diff
@@ -100,6 +100,9 @@ Features supported by Juju-owned instances running within AWS:
 
 - Juju deploys to what Amazon refers to as *m3.medium* instances by default. you
   can specify different instance types by using [constraints][constraints].
+
+- You can add a new model to a specific virtual private cloud (VPC) from the
```

Comment:
trying not to be picky, but I think it would be more correct to say  "you can create a new model in a specific..."